client/components/game/home/advisorPanel.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+var testList = [{
+  name:"Dane",
+  dialog:"Business is booming!",
+},
+  {
+    name:"Ashley",
+    dialog:"Can't we produce less waste somehow?",
+  },
+  {
+    name:"Gary",
+    dialog:"Oi! YOU'RE DRIVING US INTO THE GROUND!!"
+  }
+];
+export default React.createClass({
+  testList,
+  render() {
+    return (
+        <div>
+          <h2>Your Advisors</h2>
+          <ul>
+            {this.testList.map(function(listvalue){
+              return <li>{listvalue.name} - {listvalue.dialog}</li>;
+            })}
+          </ul>
+        </div>
+    );
+  }
+});
```

Comment:
This is a good first pass.

Two things to do:

1. Move that data into the `data` folder.
2. Assume that data comes from a property instead (ie: `this.props.advisors`).
3. Move the `<li>...</li>` into a separate React file called something like `advisor-item`.